fbgemm_gpu/test/test_utils.py

```diff
@@ -182,3 +186,15 @@ def cpu_and_maybe_gpu() -> st.SearchStrategy[List[torch.device]]:
 
 def cpu_only() -> st.SearchStrategy[List[torch.device]]:
     return st.sampled_from([torch.device("cpu")])
+
+def skipIfRocm(reason="test doesn't currently work on the ROCm stack"):
+    def skipIfRocmDecorator(fn):
+        @wraps(fn)
+        def wrapper(*args, **kwargs):
+            if TEST_WITH_ROCM:
+                raise unittest.SkipTest(reason)
+            else:
+                fn(*args, **kwargs)
+        return wrapper
+    return skipIfRocmDecorator
+    
```

Comment:
nit: I believe the problem is whitespace before the newline. Can you remove it?

Reply:
Yeah. It's removed.